<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintPopover</class>
 <widget class="QWidget" name="PrintPopover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string notr="true">Print</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="printerLayout">
     <item>
      <widget class="QWidget" name="containerWidget" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="1">
         <widget class="QComboBox" name="printerBox"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QRadioButton" name="trackListingButton">
          <property name="text">
           <string>Track Listing</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="pageSizeBox"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Copies</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PRINT OPTIONS</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PRINT TYPE</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="0" colspan="2">
         <widget class="QPushButton" name="printButton">
          <property name="text">
           <string>Print</string>
          </property>
          <property name="icon">
           <iconset theme="document-print"/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QRadioButton" name="jewelCaseButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Jewel Case Insert</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Printer</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="copiesBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Page Size</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QCheckBox" name="grayscaleBox">
          <property name="text">
           <string>Print in Grayscale</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QCheckBox" name="duplexBox">
          <property name="text">
           <string>Double Sided</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
